docker/compose/faucet-validator-fullnode/validator_node.yaml

```diff
@@ -0,0 +1,173 @@
+---
```

Comment:
probably only define overrides but I would prefer we figure out how to make this work within the single node testnet code path